scripts/install

```diff
@@ -104,159 +48,61 @@ ynh_app_setting_set --app=$app --key=virtualisation --value=$virtualisation
 #=================================================
 ynh_script_progression --message="Storing installation settings..."  --weight=1
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
 ynh_app_setting_set --app=$app --key=rpc_secret --value=$rpc_secret
-ynh_app_setting_set --app=$app --key=datadir --value=$datadir
 ynh_app_setting_set --app=$app --key=bootstrap_peers --value=$bootstrap_peers
 ynh_app_setting_set --app=$app --key=weight --value=$weight
 
-
-#=================================================
-# STANDARD MODIFICATIONS
-#=================================================
-# FIND AND OPEN A PORT
-#=================================================
-ynh_script_progression --message="Finding available ports..."  --weight=1
-
-### Use these lines if you have to open a port for the application
-### `ynh_find_port` will find the first available port starting from the given port.
-### If you're not using these lines:
-###		- Remove the section "CLOSE A PORT" in the remove script
-
-# Find an available port
-
-port=$(ynh_find_port --port=4000)
-ynh_app_setting_set --app=$app --key=port --value=$port
-port_api=$(ynh_find_port --port=5000)
-ynh_app_setting_set --app=$app --key=port_api --value=$port_api
-
-
-# Optional: Expose this port publicly
-# (N.B.: you only need to do this if the app actually needs to expose the port publicly.
-# If you do this and the app doesn't actually need you are CREATING SECURITY HOLES IN THE SERVER !)
-
-# Open the port
- ynh_script_progression --message="Configuring firewall..."  --weight=1
- ynh_exec_warn_less yunohost firewall allow TCP $port
-
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
 ynh_script_progression --message="Installing dependencies..."  --weight=1
 
-### `ynh_install_app_dependencies` allows you to add any "apt" dependencies to the package.
-### Those deb packages will be installed as dependencies of this package.
-### If you're not using this helper:
-###		- Remove the section "REMOVE DEPENDENCIES" in the remove script
-###		- Remove the variable "pkg_dependencies" in _common.sh
-###		- As well as the section "REINSTALL DEPENDENCIES" in the restore script
-###		- And the section "UPGRADE DEPENDENCIES" in the upgrade script
 if [ "$virtualisation" = "true" ]
 then
 	ynh_install_app_dependencies $pkg_dependencies_virtualisation
 fi 
```

Comment:
Ideally this should be handled from the manifest, in the `packages_from_raw_bash` key (I don't remember the exact name) ... ideally using the proper `systemd-detect-virt` check rather than `which modprobe`, assuming that's what is intended